knockout/index.d.ts

```diff
@@ -587,6 +587,12 @@ interface KnockoutStatic {
     /////////////////////////////////
 
     tasks: KnockoutTasks;
+    
+    /////////////////////////////////
+    // utils.js
+    /////////////////////////////////
+
+    onError: (error: Error) => any;
```

Comment:
`onError?: (error: Error) => void;`
- it is optional, since may or may not be defined;
- returns `void` since nothing expected to be returned.